common/chat/helper.ts

```diff
@@ -118,6 +120,35 @@ const getMembersForSubcourseGroupChat = async (subcourse: Subcourse) => {
     return members;
 };
 
+const getMatcheeConversation = async (matchees: { studentId: number; pupilId: number }): Promise<{ conversation: Conversation; conversationId: string }> => {
+    const student = await getStudent(matchees.studentId);
+    const pupil = await getPupil(matchees.pupilId);
+    const studentUser = userForStudent(student);
+    const pupilUser = userForPupil(pupil);
+    const conversationId = getConversationId([studentUser, pupilUser]);
+    const conversation = await getConversation(conversationId);
+    return { conversation, conversationId };
+};
+
+const checkChatMembersAccessRights = (conversation: Conversation): { readWriteMembers: string[]; readMembers: string[] } => {
+    const readWriteMembers: string[] = [];
+    const readMembers: string[] = [];
+
+    for (const participantId in conversation.participants) {
+        const participant = conversation.participants[participantId];
+        const access = participant.access;
+
+        if (access === 'ReadWrite') {
+            readWriteMembers.push(participantId);
+        } else if (access === 'Read') {
+            readMembers.push(participantId);
+        } else {
+            console.log(`Teilnehmer mit der ID ${participantId} hat unbekannte Zugriffsrechte.`);
```

Comment:
I'd prefer to throw an error here, unknown access control is usually a bad thing